web/package.json

```diff
@@ -94,15 +94,17 @@
     "@craco/craco": "^7.1.0",
     "@cypress/code-coverage": "^3.10.0",
     "@cypress/instrument-cra": "^1.4.0",
+    "@eslint/compat": "^1.1.1",
     "@testing-library/dom": "^10.1.0",
     "@testing-library/jest-dom": "^6.4.2",
     "@testing-library/react": "^16.0.0",
     "@testing-library/user-event": "^14.5.2",
     "@types/jest": "^29.5.12",
     "@types/recharts": "^1.8.23",
     "cypress": "^13.0.0",
-    "eslint": "^8.12.0",
-    "eslint-plugin-react-hooks": "^4.4.0",
+    "eslint": "^9.7.0",
+    "eslint-plugin-react-hooks": "^4.6.2",
+    "globals": "^15.8.0",
```

Comment:
the eslint config file is added by a npx command. and then it uses some old version of globals that's floating around from other packages. which has a white space error that now gets linted so i manually installed this version. seems silly but...?

Reply:
Yeah that's fine, thanks for catching this too.